pkg/config/config.go

```diff
@@ -696,6 +696,21 @@ func (c *Config) FindConmon() (string, error) {
 		c.Libpod.ConmonPath)
 }
 
+func (c *Config) GetDefaultEnv() []string {
+	var env []string
+	if c.Containers.EnvHost {
+		env = append(env, os.Environ()...)
+	} else if c.Containers.HTTPProxy {
```

Comment:
The behavior that those options are exclusive is not mentioned in the man page. Do we really need this? I would expect the `http_proxy` option to overwrite the host variables.

Reply:
http_proxy is gotten from the host in either case.  If user sets EnvHost, then entire host environment is passed into container, if the user specifies HTTPProxy then only the http*_proxy environments from the host are passed in.

Reply:
Makes sense.